sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -474,6 +540,308 @@ await using (var consumer = client.CreateConsumer(EventHubConsumer.DefaultConsum
+                        Assert.That(receivedEvents.Single().IsEquivalentTo(stampEvent), Is.True, "The received event did not match the sent event.");
+
+                        // Next receive on this partition shouldn't return any more messages.
+
+                        Assert.That(await consumer.ReceiveAsync(10, TimeSpan.FromSeconds(2)), Is.Empty);
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        ///   Verifies that the <see cref="EventHubConsumer" /> is able to
+        ///   connect to the Event Hubs service and perform operations.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ConsumerCanReceiveFromEarliestEvent()
+        {
+            await using (var scope = await EventHubScope.CreateAsync(1))
+            {
+                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
+
+                await using (var client = new EventHubClient(connectionString))
+                {
+                    var partition = (await client.GetPartitionIdsAsync()).First();
+
+                    await using (var consumer = client.CreateConsumer(EventHubConsumer.DefaultConsumerGroupName, partition, EventPosition.Earliest))
+                    await using (var producer = client.CreateProducer(new EventHubProducerOptions { PartitionId = partition }))
+                    {
+                        // Sending some events beforehand so the partition has some information.
+
+                        for (int i = 0; i < 10; i++)
+                        {
+                            await producer.SendAsync(new EventData(new byte[1]));
+                        }
+
+                        // Receive and validate the events; because there is some non-determinism in the messaging flow, the
+                        // sent events may not be immediately available.  Allow for a small number of attempts to receive, in order
+                        // to account for availability delays.
+
+                        var receivedEvents = new List<EventData>();
+                        var index = 0;
+
+                        while ((receivedEvents.Count < 10) && (++index < ReceiveRetryLimit))
+                        {
+                            receivedEvents.AddRange(await consumer.ReceiveAsync(20, TimeSpan.FromMilliseconds(25)));
+                        }
+
+                        Assert.That(receivedEvents.Count, Is.EqualTo(10), "The number of received events should be 10.");
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        ///   Verifies that the <see cref="EventHubConsumer" /> is able to
+        ///   connect to the Event Hubs service and perform operations.
+        /// </summary>
+        ///
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        [Ignore("isInclusive parameter is not supported yet")]
```

Comment:
Offsets will never have the inclusive flag.   They'll always be inclusive because they're predictable and sequential.   If you'd like the next event, you can just specify the next integer in sequence.

Reply:
I assumed it would keep this Track One functionality, but disabling it makes sense. I'll update the test accordingly.